### DISCUSSION TITLE
Loss of PNG transparencies

### DISCUSSION THREAD

**Discussion by JohnCover** [ValentinH/react-easy-crop#339] [Q&A] [answered]
Hey there, when I upload a PNG with a transparent background the resulting file loses its transparency; defaulting to a black background.

Is it possible to avoid this?

![image](https://user-images.githubusercontent.com/16791747/147860249-80bb6309-83bc-4a93-8d1c-6e6db2b3283f.png)

**Comment by ValentinH** [comment, marked as answer]
I think you can check this issue: https://github.com/ricardo-ch/react-easy-crop/issues/308

**Reply by JohnCover** [reply]
Thanks @ValentinH!
